aten/src/ATen/native/quantized/cpu/ACLUtils.h

```diff
@@ -0,0 +1,524 @@
+#pragma once
+
+#include <ATen/Config.h>
+#if AT_MKLDNN_ACL_ENABLED()
+
+#include <ATen/Parallel.h>
+#include <ATen/native/quantized/cpu/OnednnUtils.h>
+#include <arm_compute/core/Error.h>
+#include <arm_compute/core/TensorInfo.h>
+#include <arm_compute/core/Utils.h>
+#include <arm_compute/core/utils/quantization/AsymmHelpers.h>
+#include <arm_compute/function_info/ActivationLayerInfo.h>
+#include <arm_compute/runtime/Allocator.h>
+#include <arm_compute/runtime/NEON/functions/NEActivationLayer.h>
+#include <arm_compute/runtime/NEON/functions/NEGEMMLowpMatrixMultiplyCore.h>
+#include <arm_compute/runtime/NEON/functions/NEQuantizationLayer.h>
+#include <arm_compute/runtime/Tensor.h>
+#include <array>
+
+namespace at::native::acl_utils {
+
+using ACLQuantMatmulCacheKey = std::tuple<
```

Comment:
Nit: we are already in `acl_utils` namespace, do we still want to prefix everything with `ACL`?

Reply:
Good point!
I removed the ACL prefixes from all structs inside the `at::native::acl_utils` namespace.